ptf_tests/tests/l3_action_profile_with_vhost_ports.py

```diff
@@ -119,14 +119,16 @@ def runTest(self):
                 function_dict[table['description']](table['switch'],table['name'], match_action)
 
         time.sleep(100)
-        # configuring VMs
+
+        # Configuring VMs
         print("Configuring VM0 ....")
         test_utils.configure_vm(conn1, vm1_command_list)
 
         print("Configuring VM1 ....")
         test_utils.configure_vm(conn2, vm2_command_list)
 
-        # verify whether traffic hits group-1
+        # Verify whether traffic hits Unicast traffic from VM0 to VM1
+        print("Verify whether traffic hits Unicast traffic from VM0 to VM1")
```

Comment:
Verify unicast traffic is received

Reply:
changed the comment

Reply:
is this comment about verifying unicast traffic with scapy?